pages/06.policies/09.Promoting-integrations/docs.md

```diff
@@ -0,0 +1,71 @@
+facebookenable: true
+menu: 'Promoting integrations'
+taxonomy:
+    category:
+        - docs
+---
+
+# Introduction
+
+The Mautic website serves as a hub for the Mautic community, showcasing innovative projects that align with our mission and values. The purpose of this policy is to provide clarity on the criteria and process for listing projects and integrations on the Mautic website's home page.
+
+# Criteria for Inclusion
+
+Projects featured in the Mautic Integrations Directory must be proposed by a member of Mautic via the [Community Portal](https://community.mautic.org/assemblies/marketing-team/f/53/proposals/new). Integrations require a minimum of five votes from the community to be considered for inclusion, and be endorsed by the Marketing Team. 
+
+The projects listed on the Mautic website home page are selected based on their popularity and relevance to Mautic users, which is accomplished with a quarterly vote on the Community Portal of all eligible integrations. Members of Mautic are eligible to vote. 
+
+1. Popularity:
+
+- Projects that have gained popularity within the Mautic community are more likely to be featured on the home page, as members are more likely to vote for them.
+2. Use Cases:
+
+- Projects that demonstrate a clear alignment with Mautic's use cases and goals will be given priority.
+
+- The Marketing Team will assess how well a project addresses the needs of Mautic users and enhances their experience.
+3. Members:
+
+- Projects that are members of Mautic will be given priority over those that are not.
+
+# Maintenance Responsibilities
+
+The maintenance of projects listed on the Mautic website home page is the responsibility of the Marketing Team. This ensures consistency, accuracy, and a seamless experience for visitors.
+
+1. Content Updates:
+
+- The Marketing Team will regularly review and update the information related to listed projects to reflect their current status, features, and any relevant changes.
+
+- Project maintainers are encouraged to communicate updates to the Marketing Team for timely inclusion.
+
+2. Removal of Inactive Projects:
+
+- Inactive projects, or those no longer aligning with Mautic's goals, may be removed from the home page to maintain relevance.
+
+- The Marketing Team will communicate with project maintainers before removal, providing an opportunity for clarification or updates.
+
+# Priority for Open Source Projects
+
+Projects with an open source nature are given priority for listing on the Mautic website home page. Open source aligns with the collaborative and community-driven spirit of Mautic. However, partner projects may also be considered based on their significance and value to the Mautic ecosystem.
+
+1. Open Source Priority:
+
+- Open source projects are favored for inclusion due to their accessibility, transparency, and collaborative nature.
+
+- The Marketing Team encourages the community to actively contribute to Open Source projects.
+
+2. Partner Projects:
```

Comment:
@tobsowo what do you mean by 'Partner project's here? I'm a bit concerned it might be confused with our 'Mautic Partners'. Could you maybe explain what you mean so we can word it better? Thanks!

Reply:
Partners has to do with the official community partners

Reply:
Ah awesome - I'll make sure that's worded clearer then!